examples/widgets/styled_text.dart

```diff
@@ -53,7 +53,8 @@ HAL: This mission is too important for me to allow you to jeopardize it.''';
 
   Component createSeparator() {
     return new Container(
-      constraints: const BoxConstraints.expandWidth(maxHeight: 0.0),
+      constraints: const BoxConstraints.tightFor(
+        width: double.INFINITY, height: 0.0),
```

Comment:
I think we should avoid having double.INFINITY in the call sites as much as possible, on the principle that infinities are confusing and it's an ugly constant. Keeping the use of infinity as an internal implementation detail feels nicer. As such, I think this particular one was better using a different constructor. Having said that, maybe BoxConstraints.expand(minHeight: 0.0, maxHeight: 0.0), rather than expandWidth(), would work, so we only need one expand() constructor?